### ISSUE TITLE
slashd error alert e-mail: line 165 in subscribemail.pl (status 25, signal 0)

### ISSUE THREAD

**Issue by marty-b** [SoylentNews/rehash#291]
Received multiple e-mails containing this error message:
   SoylentNews Slashd Error Alert!
   SLASHD line 165 produced the following error:
   subscribemail.pl odd exit (status 25, signal 0)
   ----------------------------------------------------------------------
   You have received this message because you subscribed to it
   on SoylentNews.  To stop receiving this and other
   messages from SoylentNews, or to add more messages
   or change your preferences, please go to your user page.

	   https://soylentnews.org/my/messages

   You can log in and change your preferences from there.

On these dates:
   2016-04-27
   2016-06-17
   2016-06-22
   2016-06-28
   2016-07-05
   2016-07-21
   2016-08-04
   2016-08-07

Seems that e-mail is sent when a subscription is extended at no cost[*],
AND there is no other *paid* subscription on that date.

See: https://soylentnews.org/admin.pl?op=recent_subs

Steps to reproduce:

Basically, grant subscription days to myself, or to someone else,
on a day when nobody has paid for a subscription.

NOTE: User needs admin rights in order to perform these steps.

Steps to grant subscription days to myself:

[*] On my preferences page:
  https://soylentnews.org/my/info

1.) Clicked on the "Subscriptions" tab for my user nick.
2.) Scrolled down to the "Grant Subscriptions" area.
3.) in the field with this prompt:
  "Want to grant yourself some subscription days? You're an admin, so grant as many as you want:"
  I entered "30" to extend my subscription for one month.
4.) Click on the [Grant] button.
5.) Hope nobody else subscribes on that same day (UTC).
